apps/examples/testcase/le_tc/kernel/itc_pthread.c

```diff
@@ -329,22 +335,16 @@ static void itc_pthread_mutex_init_destroy_p_multitime(void)
 * Preconditions         :pthread_key_create
 * Postconditions        :none
 */
-static void itc_pthread_key_create_set_getspecific_p_multitime(void)
+static void itc_pthread_key_create_set_get_specific_p_multitime(void)
 {
 	int ret_chk;
 	g_flag = false;
-	g_key = 0;
-
-	ret_chk = pthread_key_create(&g_key, NULL);
-	TC_ASSERT_EQ("pthread_key_create", ret_chk, OK);
 
 	ret_chk = pthread_create(&g_pthread, NULL, thread_set_get_specific, NULL);
```

Comment:
thread_set_get_specific should be changed to include create and delete.
I think itc_pthread_key_test is enough.

Reply:
changed subroutine name as you mentioned